onnxruntime/core/common/cpuid_info.h

```diff
@@ -5,13 +5,18 @@
 
 #include "core/common/common.h"
 #include "core/common/cpuid_arch_definition.h"
+#include <mutex>
 
 namespace onnxruntime {
 
 class CPUIDInfo {
  public:
   static const CPUIDInfo& GetCPUIDInfo() {
     static CPUIDInfo cpuid_info;
+
```

Comment:
It has no difference to the old one. function local static variables already have the call_once semantic. Their initialization is thread-safe.

Reply:
You are right. Reverted it.